packages/react-scripts/template/README.md

```diff
@@ -598,6 +598,13 @@ This will allow you to do imports like
 
 > **Note:** You must prefix imports from `node_modules` with `~` as displayed above.
 
+> **Note:** If you're using Flow, add the following to your .flowconfig so it'll recognize the .scss import.
```

Comment:
Include `.sass` as well, and wrap these in the inline code back ticks please: \`.scss\`.

Can you please make the code snippet *inside* of the quote?